app/scripts/contentscript.js

```diff
@@ -545,9 +517,6 @@ const start = () => {
   }
 
   if (shouldInjectProvider()) {
-    if (!isManifestV3) {
-      injectScript(inpageBundle);
-    }
```

Comment:
`inpage.js` has been added to manifest v2 (only) as a `content_script`, so we don't need the MV3 check at all now.